RELEASE.md

```diff
@@ -1,3 +1,59 @@
+# Release 2.3.0
+
+The 2.3 minor series tracks TensorFlow 2.3.
+
+## Features
+
+- The 30 sec default reload period is now customizable in the Settings UI (#2794)
+- 🧪 **Experimental** Debugger V2 is now available; see the [tutorial][debugger-v2-tutorial] on how to use the experimental TensorFlow APIs to spot NaN's in tensors, view graphs generated from executions, and the related lines in the Python source code (#3821)
```

Comment:
nit (throughout): This document wraps at 80.

Reply:
Done.